src/Sculpin/Core/Generator/GeneratorManager.php

```diff
@@ -95,28 +95,28 @@ public function generate(SourceInterface $source, SourceSet $sourceSet)
 
         $generators = array();
         $isGenerator = $source->isGenerator();
-        if ($generatorNames = $data->get('generator')) {
```

Comment:
How much of this was stuff you did by hand versus PSR-2 automatically fixing this up? There is a lot of logic that changed in here and I'm afraid I can't follow it well enough to know if it is still correct after these changes.